.travis.yml

```diff
@@ -71,6 +71,7 @@ jobs:
                 - composer info -D | sort
             script:
                 - vendor/bin/phpunit || travis_terminate 1
+                - dev-tools/vendor/bin/phpstan analyse --level=0 --configuration=phpstan.neon src tests
```

Comment:
Can we have here only `dev-tools/vendor/bin/phpstan analyse`? Configuration path is not needed (we would benefit from `phpstan.neon`/`phpstan.neon.dist` pair) and level and paths can be move to config file.

Reply:
Done.